ajax/libs/jqgrid/package.json

```diff
@@ -1,21 +1,48 @@
 {
   "name": "jqgrid",
   "filename": "js/jquery.jqGrid.min.js",
-  "version": "4.6.0",
+  "version": "5.0.1",
   "description": "jqGrid is an Ajax-enabled JavaScript control that provides solutions for representing and manipulating tabular data on the web.",
   "homepage": "http://www.trirand.com/blog/",
   "keywords": [
+    "jQuery",
+    "jqGrid",
+    "JavaScript",
     "grid",
-    "jquery"
+    "table",
+    "pivot",
+    "tree",
+    "subgrid",
+    "sorting",
+    "paging",
+    "editing",
+    "grouping",
+    "searching",
+    "ajax"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "Creative Commons Attribution-Non-Commercial 3.0 License",
   "repository": {
     "type": "git",
-    "url": "https://github.com/tonytomov/jqGrid"
+    "url": "https://github.com/tonytomov/jqGrid/tree/master"
   },
   "author": {
     "name": "Tony Tomov",
     "url": "http://www.guriddo.net",
     "email": "tony@trirand.com"
-  }
+  },
+  "npmName": "jqgrid",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/ui.jqgrid.css",
+        "css/ui.jqgrid-bootstrap",
+        "css/addons/ui.multiselect.css",
+        "js/jquery.jqGrid.min.js",
+        "js/jquery.jqGrid.js",
+        "js/addons/ui.multiselect.js",
+        "js/i18n/*.js"
+      ]
```

Comment:
Would you like to follow the https://github.com/cdnjs/cdnjs/pull/6007#issuecomment-318597375 comment as modify from A to B?
We usually use the pattern matcher allowing selection of multiple files, see [node-glob](https://github.com/isaacs/node-glob#glob-primer)

- A 
```js 
      "files": [
        "css/ui.jqgrid.css",
        "css/ui.jqgrid-bootstrap",
        "css/addons/ui.multiselect.css",
        "js/jquery.jqGrid.min.js",
        "js/jquery.jqGrid.js",
        "js/addons/ui.multiselect.js",
        "js/i18n/*.js"
      ]
```


- B
```js
      "files": [
          "css/**/!(ui.multiselect).+(css|map)",
          "js/**/!(jqDnR|jqModal|ui.multiselect).+(js|map)",
          "plugins/**/!(jquery.contextmenu|jquery.searchFilter|jquery.tablednd).+(js|css)"
      ]
```

Thank you.